react/wishList.css

```
@@ -1,21 +1,62 @@
+:global(.vtex-lists-page .vtex-store__template) {
```

does this work? shouldn't it be
```suggestion
:global(.vtex-lists-page) :global(.vtex-store__template) {
```

---

Yes, it works :thinking: